Tasks/ExtractFiles/extractFiles.ts

```diff
@@ -0,0 +1,307 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
```

Comment:
Paths are normalized already, and the case comparison is undesired in the section you mentioned because that is very specific to Linux where case matters.